packages/utils/src/normalize.ts

```diff
@@ -1,15 +1,25 @@
 /**
- * Normalize a value between a range
+ * The `normalize` function scales a value within a specific range to a
+ * normalized range between `0` and `1`. Given an input value and a range
+ * (`min` to `max`), it calculates the proportion of the value relative to the
+ * range. This is useful for standardizing data, making it easier to work with
+ * values across different scales.
  *
- * @param value - The value to normalize
- * @param min - The minimum value of the range
- * @param max - The maximum value of the range
- * @returns The normalized value
+ * @param value - The number to be normalized within the range.
+ * @param min - The minimum value of the range.
+ * @param max - The maximum value of the range.
+ * @returns The normalized value between `0` and `1`.
  *
  * @example
  * ```ts
- * normalize(5, 0, 10) // 0.5
- * normalize(5, 0, 5) // 1
+ * // Normalize a value within the range [0, 100]
+ * normalize(50, 0, 100)); // 0.5
+ * // Normalize a value in a custom range [10, 20]
+ * normalize(15, 10, 20)); // 0.5
+ * // Normalize a value below the minimum (the function doesn't clamp)
+ * normalize(5, 10, 20)); // -0.5
+ * // Normalize a value above the maximum
+ * normalize(25, 10, 20)); // 1.5
```

Comment:
**nitpick:** Write "normalize a number" instead

Reply:
Updated copy elsewhere to `value` as this text is a descriptive text and not directly tied to it's ts type.